pkgs/applications/editors/neovim/utils.nix

```diff
@@ -226,23 +226,6 @@ let
       ''
         mkdir -p "$out/parser"
         ln -s "${grammar}/parser" "$out/parser/${name}.so"
-
-        mkdir -p "$out/queries/${name}"
-        if [ -d "${grammar}/queries/${name}" ]; then
-          echo "moving queries from neovim queries dir"
-          for file in "${grammar}/queries/${name}"*; do
-            ln -s "$file" "$out/queries/${name}/$(basename "$file")"
-          done
-        else
-          if [ -d "${grammar}/queries" ]; then
-            echo "moving queries from standard queries dir"
-            for file in "${grammar}/queries/"*; do
-              ln -s "$file" "$out/queries/${name}/$(basename "$file")"
-            done
-          else
-            echo "missing queries for ${name}"
-          fi
```

Comment:
If neovim isnt finding all of the ones packaged like this, we could leave this part in, assuming simply removing lines 231-236 fixes the issue

Id have to fix nixCats to accomodate but I have a plan for that which is backwards compatible for me and not an issue